addons/binding/org.openhab.binding.spotify/README.md

```diff
@@ -0,0 +1,225 @@
+1. The bridge thing will stay in state _INITIALIZING_ and eventually go _OFFLINE_ - this is fine. You have to authorize this bridge with Spotify.
+1. Go to the authorization page of your server. `http://<your openHAB address>:8080/connectspotify`. Your newly added bridge should be listed there.
+1. Press the _"Authorize Player"_ button. This will take you either to the login page of Spotify or directly to the authorization screen. Login and/or authorize the application. If the Redirect URIs are correct you will be returned and the entry should show you are authorized with you Spotify user name/id. If not, go back to your Spotify Application and ensure you have the right Redirect URIs.
+1. The binding will be updated with a refresh token and go ONLINE. The refresh token is used to re-authorize the bridge with Spotify Connect Web API whenever required.
+
+Now that you have got your bridge _ONLINE_ it is time to discover your devices! Go to Paper UI Inbox and search for Spotify Connect Devices. Any device currently available on your account should show up immediately.
+
+If no devices show up you can start Spotify App on your PC/Mac/iOS/Android and start playing on your devices as you run discovery.
+This should make any Spotify Connect devices and Spotify Apps discoverable.
+You may have to trigger the openHAB discovery several times as bridge will only find active devices known by the Spotify Web API at the time the discovery is triggered.
+
+Should the bridge configuration be broken for any reason, the authorization procedure can be reinitiated from step 6 whenever required.
+You can force reinitialization by authorize on the connect Spotify page if the page would show it as authorized. This will reset the refresh token.
+
+The following configuration options are available on the Spotify Bridge player:
+| Parameter     | Description                                                                                                                                                   |
+|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| clientId      | This is the Client ID provided by Spotify when you add a new Application for openHAB to your Spotify Account. Go to https://developer.spotify.com/ (Required) |
+| clientSecret  | This is the Client Secret provided by Spotify when you add a new Application for openHAB to your Spotify Account.   (Required)                                |
+| refreshPeriod | This is the frequency of the polling requests to the Spotify Connect Web API in seconds.                                                                       |
+
+The following configuration option is available on the Spotify device:
+| Parameter | Description                                           |
```

Comment:
I think that there should also be a newline before a table, please check the SAT findings.

Reply:
O, I might be wrong then 🤔